src/connectedvmware/azext_connectedvmware/_params.py

```diff
@@ -108,6 +108,20 @@ def load_arguments(self, _):
     with self.argument_context('connectedvmware vm-template delete') as c:
         c.argument('force', action='store_true', help="Whether force delete or not.")
 
+    with self.argument_context('connectedvmware vm link-to-vcenter') as c:
```

Comment:
we can think more on the naming, we can also name this as
az connectedvmware vm create-from-machine, it will be symmetrical to az connectedvmware vm create, az connectedvmware vm create-from-inventory etc and possbiliy other create variations that might come up in teh future?

Reply:
Have updated the name to `create-from-machines` as per offline discussion